### PR TITLE
modify types.Receipt.Status from string to int

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

File: client/api_handler.go [hunk 1]
```diff
@@ -351,11 +350,23 @@ func (api *APIHandler) GetTransactionByBlockNumberAndIndex(ctx context.Context,
...
+		types.Receipt
+		Status string `json:"status"`
+	}{}
+	err := api.CallContext(ctx, anonymityReceipt, "getTransactionReceipt", groupID, txhash)
 	if err != nil {
 		return nil, err
 	}
-	// js, err := json.MarshalIndent(raw, "", indent)
+	if len(anonymityReceipt.Status) < 2 {
+		return nil, errors.New("receipt is no existence")
```
Comment (bxq2011hust) [inline client/api_handler.go:362]:
```suggestion
		return nil, fmt.Errorf("transaction is not on-chain")
```

File: client/api_handler.go [hunk 2]
```diff
@@ -91,7 +92,7 @@ func (api *APIHandler) Call(ctx context.Context, groupID int, msg ethereum.CallM
 //
 // If the transaction was a contract creation use the TransactionReceipt method to get the
 // contract address after the transaction has been mined.
-func (api *APIHandler) SendRawTransaction(ctx context.Context, receipt *types.Receipt, groupID int, tx *types.Transaction) error {
+func (api *APIHandler) SendRawTransaction(ctx context.Context, receipt interface{}, groupID int, tx *types.Transaction) error {
```
Comment (bxq2011hust) [inline client/api_handler.go:95]:
```suggestion
func (api *APIHandler) SendRawTransaction(ctx context.Context, groupID int, tx *types.Transaction) (*types.Receipt, error) {
```

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- client/api_handler.go  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5913834&bid=19376027)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- client/api_handler.go  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5913834&bid=19376027)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- client/api_handler.go  5
         

Clones added
============
- client/api_handler.go  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5913834&bid=19376027)

**Pull request merged**